tabular/tests/unittests/models/test_tabular_nn.py

```diff
@@ -25,3 +26,35 @@ def test_tabular_nn_regression(fit_helper):
     )
     dataset_name = 'ames'
     fit_helper.fit_and_validate_dataset(dataset_name=dataset_name, fit_args=fit_args)
+
+
+def test_tabular_nn_binary_compile_onnx(fit_helper):
+    fit_args = dict(
+        hyperparameters={TabularNeuralNetTorchModel: {}},
+    )
+    dataset_name = 'adult'
+    compiler_configs = {TabularNeuralNetTorchModel: {'compiler': 'onnx'}}
+    fit_helper.fit_and_validate_dataset(dataset_name=dataset_name, fit_args=fit_args,
+                                        compile_models=True, compiler_configs=compiler_configs)
+
+
+def test_tabular_nn_multiclass_compile_onnx(fit_helper):
+    fit_args = dict(
+        hyperparameters={TabularNeuralNetTorchModel: {}},
+    )
+    dataset_name = 'covertype_small'
+    compiler_configs = {TabularNeuralNetTorchModel: {'compiler': 'onnx'}}
+    fit_helper.fit_and_validate_dataset(dataset_name=dataset_name, fit_args=fit_args,
+                                        compile_models=True, compiler_configs=compiler_configs)
+
+
+@pytest.mark.skip(reason="FunctionTransformer is not supported unless the transform function is None (= identity).")
```

Comment:
This doesn't need to skipped anymore right?

Reply:
Actually, we still need to skip this, since we don't support converting `OneHotMergeRaresHandleUnknownEncoder` to onnx yet. It is required to handle columns with `onehot_features`. I think we can address this in a separate PR.

Reply:
so currently it will crash if someone calls compile with NN_TORCH on certain datasets? That is something we should be careful about.

Reply:
That's a good point. I just added the support to the onehot encoder, so that 1) we don't need to skip this test case, 2) there shouldn't be a crash if someone calls compile with NN_TORCH on certain datasets.